### PR TITLE
Vendor tini

### LINKED ISSUE #648: Cache or vendor `tini`
We should consider caching or vendoring `tini` as we got an error like this:
image

### PR DESCRIPTION
**Pull request by e00E**
fixes #648

It seems like my editor also automatically removed a few trailing whitespaces. Imo we can just keep that in the commit.

### CODE REVIEW

File: docker/rust/Dockerfile [hunk 1]
```diff
@@ -38,7 +38,10 @@ ONBUILD COPY target/debug/driver stablex
 FROM ${RUST_BASE}
 
 # Add Tini
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+# We ran into github rate limiting using this url so we now keep a local copy of
+# tini instead.
+# ENV TINI_VERSION v0.18.0
+# ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+Add docker/rust/tini /tini
```
Comment (fleupold) [inline docker/rust/Dockerfile:45]: ubernit: ADD instead of Add

File: docker/rust/Dockerfile [hunk 2]
```diff
@@ -38,7 +38,10 @@ ONBUILD COPY target/debug/driver stablex
 FROM ${RUST_BASE}
 
 # Add Tini
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+# We ran into github rate limiting using this url so we now keep a local copy of
+# tini instead.
+# ENV TINI_VERSION v0.18.0
+# ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
```
Comment (fleupold) [inline docker/rust/Dockerfile:44]: Not sure if it's worth keeping the old command around, but doesn't matter much.

**Review by fleupold — APPROVED** [review]

**Review by bh2smith — APPROVED** [review]

File: docker/rust/Dockerfile (hunk 2 above)
Comment (bh2smith) [reply docker/rust/Dockerfile:44]: I think its nice to know where it came from.

Comment (nlordell) [reply docker/rust/Dockerfile:44]: I would rather add this information to `docker/rust/README.md`

**Review by nlordell — APPROVED** [review]

File: docker/rust/Dockerfile (hunk 2 above)
Comment (e00E) [reply docker/rust/Dockerfile:44]: I prefer it in this place because it right at the position where it is relevant so I'll leave it as is for now.

**Pull request merged**